### ISSUE TITLE
Twitter account

### ISSUE THREAD

**Issue by Changaco** [AspenWeb/salon#18]
Hey @chadwhitacre, do you still have access to the [@aspen_io Twitter account](https://twitter.com/aspen_io)? It looks like that account was never transferred or shared, so you're probably the only person who has access it.

**Comment by chadwhitacre** [comment]
Yeah, I have it. I'll email you the login and you can update the email address, etc.
